src/nnet3/nnet-test-utils.cc

```diff
@@ -793,7 +793,7 @@ void ComputeExampleComputationRequestSimple(
 
 static void GenerateRandomComponentConfig(std::string *component_type,
                                           std::string *config) {
-  int32 n = RandInt(0, 20);
+  int32 n = RandInt(0, 21);
```

Comment:
Yes-- not to check in.

On Tue, Nov 3, 2015 at 2:07 PM, hhadian <notifications@github.com> wrote:

> In src/nnet3/nnet-test-utils.cc
> <https://github.com/kaldi-asr/kaldi/pull/307#discussion_r43791786>:
>
> > @@ -793,7 +793,7 @@ void ComputeExampleComputationRequestSimple(
> >
> >  static void GenerateRandomComponentConfig(std::string *component_type,
> >                                            std::string *config) {
> > -  int32 n = RandInt(0, 20);
> > +  int32 n = RandInt(0, 21);
>
> Do you mean temporarily to test the testing codes ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/307/files#r43791786>.
>